webdriver/state/text/get_text.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+import uuid
+
+from support.asserts import assert_error, assert_success
+from support.inline import inline
+
+def test_getting_text_of_a_non_existant_element_is_an_error(session):
+   session.url = inline("""<body>Hello world</body>""")
+   id = uuid.uuid4()
+
+   result = session.transport.send(
+       "GET",
+       "session/%s/element/%s/text" % (session.session_id, id))
```

Comment:
I'm not sure I agree with that, although I see the logic. And I don't think there's much precedent for actaully doing that. But I guess it's fine. A comment would be nice.